hmpps-prisoner-search-indexer/src/main/kotlin/uk/gov/justice/digital/hmpps/prisonersearch/indexer/model/translator.kt

```diff
@@ -148,3 +151,12 @@ private fun IncentiveLevel?.toCurrentIncentive(): CurrentIncentive? = this?.let
     dateTime = it.iepTime.withNano(0),
   )
 }
+
+private fun List<BodyPartDetail>?.addIfContains(bodyPart: BodyPartDetail, keyword: String): List<BodyPartDetail>? =
```

Comment:
how about
```suggestion
private fun List<BodyPartDetail>?.addIfCommentContains(bodyPart: BodyPartDetail, keyword: String): List<BodyPartDetail>? =
```
I struggled with the function initially - thought it was a built in and didn't realise it was one of ours